docs/static/openapi.yml

```diff
@@ -46480,7 +46480,7 @@ paths:
                           service:
                             title: >-
                               The Service for which the application is
-                              configured for
+                              configured
```

Comment:
This file always changes for me - am I the only one? I usually just `stash` it.

Reply:
tl;dr Don't worry too much about it. 

It gets regenerated with `ignite generate openapi --yes` and we don't have anything in place to keep good track of it atm.